api/spec/src/billing/pending.tsp

```diff
@@ -0,0 +1,51 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter.Billing;
+
+@route("/api/v1/billing/pending")
+@tag("Billing (Experimental)")
+interface PendingCharges {
```

Comment:
TODO: let's generate an invoice initially so that we don't have to have a separate api